test/test_transforms.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import division
 import os
+import mock
```

Comment:
I was fiddling with patching `PILLOW_VERSION` in `torchvision.transforms.functional` in order to test the handling when `pillow<5.2.0`. As far as I understand it, you can only patch constants before the module is loaded. If we want to do that we need to do it around here. Do we want to also test this?

Reply:
I'd say it's ok if we don't test this case, and just assume that pillow < 5.2.0 is a best-effort support